src/vsr/replica.zig

```diff
@@ -9477,7 +9548,7 @@ pub fn ReplicaType(
             // could lead to permanent unavailability.
```

Comment:
The comment needs adjustment, as we now use "prepared atop" rather than "committed atop", though perhaps there's a better phrasing here.

Reply:
Good catch!

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2104/files#diff-e95e4dc6c1aae349e798cfeed22ad01f830cce0e654e6b4b765b3e7b0c677b6aR9525-R9532